src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateSplit = date.split(fromFormat[3]);
+  const newDate = [];
+
+
+  for (let i = 0; toFormat.length - 1 > i; i++) {
```

Comment:
The loop condition `toFormat.length - 1 > i` will cause the loop to miss the last element in the `toFormat` array. It should be `i < toFormat.length - 1` or simply `i < toFormat.length` if you want to iterate through all elements.